File: js/easy/buildEgyptTriangle.js
function print(val) {
  console.log(val);
}

function generateTriangle() {
  var cell = '#';
  var result = '';
  for (var i = 1; i <= 7; i++) {
    result += cell.repeat(i) + '\n';
  }
  print(result);
}

function main() {
  generateTriangle();
}
main();

File: dart/easy/buildEgyptTriangle.dart
class BuildEgyptTriangle {
  static String result(int n) {
    String cell = '#';
    String result = '';
    if (n < 1) {
      return result;
    }
    for (int i = 1; i <= n; i++) {
      result += cell * i + '\n';
    }
    return result;
  }
}
